climada/hazard/tc_tracks.py

```diff
@@ -515,7 +515,7 @@ def _read_file_emanuel(self, path, hemisphere='S', rmw_corr=False):
                     hours[i_track, valid_idx[reference_idx]],)
                 datetimes = [reference_date + dt.timedelta(hours=int(step * i))
                              for i in range(nnodes)]
-            datetimes = np.array(datetimes)
+            datetimes = np.array(datetimes, dtype=np.datetime64)
```

Comment:
This is also unrelated to the main topic of this PR: I noticed along the way that this array's `dtype` would be automatically set to `object` (instead of the more natural `np.datetime64`) which might cause problems in the future.